spec/jobs/rapid_ready_for_decision/rrd_processor_spec.rb

```diff
@@ -22,6 +24,30 @@
     end
   end
 
+  describe '#assess_data_with_vro' do
+    subject { rrd_processor.assess_data_with_vro }
+
+    it 'calls VRO and extracts the returned evidence' do
+      allow(rrd_processor).to receive(:sufficient_evidence?).and_return(true)
+      VCR.use_cassette('virtual_regional_office/health_assessment') { subject }
+      expect(rrd_processor.claim_context.assessed_data).to include(:medications, :conditions, :procedures, :bp_readings)
+      expect(rrd_processor.claim_context.sufficient_evidence).to be true
+    end
+  end
+
+  describe '#generate_pdf_body_with_vro' do
+    subject { rrd_processor.generate_pdf_body_with_vro }
+
+    it 'calls VRO to generate the PDF, and again to download it' do
+      allow(rrd_processor.claim_context).to receive(:assessed_data).and_return({ medications: [] })
+      VCR.use_cassette('virtual_regional_office/evidence_pdf') do
+        VCR.use_cassette('virtual_regional_office/evidence_pdf_download') do
+          expect(subject).to start_with('%PDF-1.3')
+        end
+      end
+    end
+  end
+
```

Comment:
Can we move this to after line 93 (within the `context 'when VRO flag is enabled'`)? `assess_data_with_vro` and `generate_pdf_body_with_vro` shouldn't be called when `Flipper.disable(:rrd_call_vro_service)` (line 11), so having these tests here confused me.

Reply:
I think what I actually want to do is move the `disable` line into the top level of "when run in a Sidekiq job" because that's the one context that was failing due to the flag.

The more granular tests for `*_with_vro` methods are agnostic of the flag.